### ISSUE TITLE
10-7959f-2: Introduction 

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#78669]
As a user on Va.gov, I want to see the introduction so that I know what the form is about when I go to file my claim.

- breadcrumb
- header and subheader
- Intro text
- process steps
- sign in text and header
- respondent burden, OMB control, expiration date, etc.
- privacy act link
- intro page following [guidelines](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/content/form-introduction-page-template.md) 

[figma](https://www.figma.com/design/WYZCqWcS2gJWIqLtmQlvl7/FMP-Claim-Cover-Sheet-(10-7959f-2)?node-id=726-21596&t=zVSEtz4InLjoF2fp-0)

Acceptance criteria
- given a user on Va.gov goes to the online 10-7959f-2
- when they go to file a medical claim [page](https://www.va.gov/resources/how-to-file-a-va-foreign-medical-program-claim/) and click on the online option
- then they will land on the intro page 
- and they can fill out the rest of the form

**Comment by reiting** [comment]
This is complete and currently in team review. <img width="679" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHFlQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--378836ed6826f6ed2f04d987dcf9408299291cf2/Screenshot%202024-06-28%20at%2010.19.03%E2%80%AFAM.png" alt="Screenshot 2024-06-28 at 10.19.03 AM.png" />

**Comment by reiting** [comment]
url for f2 on staging: https://staging.va.gov/health-care/foreign-medical-program/file-claim-form-10-7959f-2

**Comment by reiting** [comment]
@marywang2 @baltes @brian-wilke this should be in staging and ready to go. See the comment above for the url.

**Comment by baltes** [comment]
Reviewing, requested access to the figma file

**Comment by baltes** [comment]
@reiting @brian-wilke Can either of you confirm that we're not missing part of the breadcrumb? It just seems short...

If it's correct, then the rest looks good to me.

<img width="1148" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNktyQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fa648186f029454e1cb229290d0dea6b1f4c7fad/image.png" alt="image.png" />

**Comment by brian-wilke** [comment]
Will review... not seeing attached image.

**Comment by baltes** [comment]
@brian-wilke it sounds like we need @reiting to take a look at changing the breadcrumb, is that correct? And i'm not sure I'm clear on expected versus actual behavior with your comment on wrap vs truncate but I'll take a look on mobile.

**Comment by baltes** [comment]
This is the breadcrumb I see on mobile for all pages I've looked at...<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNVd4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c27093a8c833c83274343e20855bf36ed4cab3f2/IMG_7411.png" alt="IMG_7411.png" />

**Comment by brian-wilke** [comment]
Although I can't see the attached image... I think we are actually in compliance.
With enough zoom (+) the breadcrumbs truncate appropriately.

**Comment by marywang2** [comment]
@reiting just making sure you saw the comments on the breadcrumb - the desktop view should reflect what's on the figma and the body copy also needs to be updated (bullet points and link should be added). Thanks!

**Comment by reiting** [comment]
@marywang2 I added the file a claim to the breadcrumbs (not yet on staging), but other than that, it seems to match Figma. The bullet points and sign in button/start without signing in button are there. Can you please clarify further what is missing? Thanks!

**Comment by marywang2** [comment]
The bullet points don't match the copy in the figma (there should also be a link): <img width="394" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOFBFQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f5e55fbd23e3e8ed982ac827af7150469fb08058/Screenshot%202024-07-09%20at%209.12.26%E2%80%AFAM.png" alt="Screenshot 2024-07-09 at 9.12.26 AM.png" />

**Comment by reiting** [comment]
@marywang2 I must have the wrong Figma link then. I used the one in the story, can you link the most updated one here?

**Comment by marywang2** [comment]
Tagging @brian-wilke  to update the annotated figma screens.

**Comment by brian-wilke** [comment]
Updates in place.

**Comment by reiting** [comment]
@marywang2 I've updated this and put a PR in for it. In team review right now. <img width="700" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEhGQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1894f4acb913891ebba24cab41c9569b5d66b7e3/Screenshot%202024-07-09%20at%2011.39.47%E2%80%AFAM.png" alt="Screenshot 2024-07-09 at 11.39.47 AM.png" />

**Comment by reiting** [comment]
This has been approved and merged and should be in staging now @marywang2

**Comment by marywang2** [comment]
I'm seeing a typo: should be "certain" rather than "cetain" in the last bullet point. (cc @brian-wilke pls review as well).<img width="1112" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOExSQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d9e14d7786e9505e6f56f4c50afc6a4cfbe5c603/Screenshot%202024-07-11%20at%2011.38.40%E2%80%AFAM.png" alt="Screenshot 2024-07-11 at 11.38.40 AM.png" />

**Comment by reiting** [comment]
<img width="621" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNGZTQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7fdd41deffff321297543abd7152f5f409552fe4/Screenshot%202024-07-11%20at%201.53.29%E2%80%AFPM.png" alt="Screenshot 2024-07-11 at 1.53.29 PM.png" />

This is in team review.

**Comment by reiting** [comment]
This is finally in platform review. There were some errors originally when pushed that were due to some problem in main. It has since been fixed so this finally passed and is waiting on platform.

**Comment by reiting** [comment]
This has been approved and merged. Should be on staging in a few hours.

**Comment by marywang2** [comment]
@reiting I'm seeing some other typos just now in the bullet copy(didn't catch them earlier): it should be "receipt" and "supporting" rather than "support". Are you copying/pasting or typing the text in?

**Comment by reiting** [comment]
Created a new PR to resolve those issues.

**Comment by marywang2** [comment]
Thanks @reiting - I want to make sure the text is being entered via copy/paste if at all possible - it seems there's a likelihood of errors being introduced by the manual entry.

**Comment by reiting** [comment]
@marywang this PR has been approved and merged. It should be ready to review in staging in a few hours.
